### ISSUE TITLE
Radiant mid laner bot returning to mid lane via Dire side of river

### ISSUE THREAD

**Issue by Riveon** [Nostrademous/Dota2-FullOverwrite#213]
I've noticed in two matches that when this script is on Radiant, after the midlaner goes to the bottom rune they return to the midlane via the dire side of the river (next to the bottom rune) and walk past the dire tower, which means they get hit by the tower and they're also vulnerable to attack from the dire midlaner.  Not sure if it happens on the top side of the map - so far I've only seen it happen in this specific location.

**Comment by CsharpNoobDude** [comment]
Blink is weird in a lot of ways - probably the same reasoning as for some abilities like Invokers Alacrity - initiating but not committing to doing it.

**Comment by Riveon** [comment]
Yep you're right. And in a lot of cases it seems like they want to retreat by blinking in the direction of their base but they end up blinking in the opposite direction.

**Comment by Riveon** [comment]
Here's a gif showing another tower evasion/avoidance issue I noticed, where CM kept getting hit by the tower: https://gfycat.com/PotableFragrantFlycatcher
